### ISSUE TITLE
Show Twitter notification save button when enabling/disabling

### ISSUE THREAD

**Issue by FirrSkunk** [owncast/owncast#2534]
### Share your bug report, feature request, or comment.

though the admin panel, you can enable and disable the twitter integration.  If you were to flip it off and later turn it back on, the settings will return and it will appear like it has been enabled, but it doesn't actually save unless you also edit one of the text blocks.  

![image](https://user-images.githubusercontent.com/59941415/210119645-5c5bacec-6a03-4b5d-a6e6-5ba2599b6ea7.png)

Currently the workaround I've found is editing the title to something else and back will show the save button.

![image](https://user-images.githubusercontent.com/59941415/210119613-b098f6ab-4327-4925-be2a-d6ad9ca6cc7b.png)

I wouldn't think it would be too hard to add the switch changing as a case to make the save button appear.

**Comment by gabek** [comment]
Thanks! You're the first person I've heard of actually using this, so it's good to know it's useful.

**Comment by hsingh124** [comment]
Hey, could I work on this issue? I'm just starting out in open source development and I think this would be a good first issue.

**Comment by gabek** [comment]
Absolutely! It should be pretty straightforward, but let me know if you have any questions. You'll find this on the `Notifications` section of the admin (`/admin`).

**Comment by hsingh124** [comment]
Awesome, thanks @gabek!

**Comment by hsingh124** [comment]
@gabek I've made a pull request for the fix. Basically just removed the dependency of `formDataValues.enabled` from the canSave() logic.

I did find another issue though. When twitter is enabled, we can save even if the required fields are empty. If we save with one of the required fields as empty and then toggle twitter off, the save button disappears. I've attached a video of the workflow below. Should there be some validation in the save method for this?

https://user-images.githubusercontent.com/44574494/210133610-5289ad56-f7a8-4be8-aefb-376c84c11732.mov
